packages/datadog-plugin-aws-sdk/test/fixtures.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
```

Comment:
Fixtures by service (different files) would be easier to work with.

Reply:
i've refactored this to split out the fixtures into their own files, lmk if the structure looks alright to you, should be easier to sort through now.